chromadb/test/property/test_embeddings.py

```diff
@@ -73,13 +79,30 @@ def __init__(self, api: ServerAPI):
     @initialize(collection=collection_st)  # type: ignore
     def initialize(self, collection: strategies.Collection):
         self.api.reset()
-        self.collection = self.api.create_collection(
-            name=collection.name,
-            metadata=collection.metadata,
-            embedding_function=collection.embedding_function,
-        )
+        try:
+            self.collection = self.api.create_collection(
+                name=collection.name,
+                metadata=collection.metadata,
+                embedding_function=collection.embedding_function,
+            )
+        except Exception as e:
```

Comment:
Whats this doing and why - seems comment-worthy

Reply:
`hnsw:batch` is only possibly persisted client/server. So, instead of making a more complex change to the test rig, this is one way to detect whether the client is persisted.